File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        longest = 0
        s = set(nums)
        for n in s:
            if n-1 in s:
                continue
            else:
                length = 1
                while n+length in s:
                    length += 1
                longest = max(longest, length)
        return longest

File: Data Structures & Algorithms/two-integer-sum/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        const visited = new Map()
        for (let i = 0; i < nums.length; i++){
            const n = nums[i]
            const difference = target-n
            if (visited.has(difference)){
                return [visited.get(difference), i]
            }
            visited.set(n, i)
        }
    }
}
